lib/mimic/server.ex

```diff
@@ -402,6 +402,9 @@ defmodule Mimic.Server do
     case :ets.lookup(__MODULE__, {owner_pid, module}) do
       [{{^owner_pid, ^module}, actual_owner_pid}] ->
         :ets.insert(__MODULE__, {{allowed_pid, module}, actual_owner_pid})
+
+      [] ->
```

Comment:
Hmm I think this just hides the problem.

I tried changing this test to `allow` before expect/stub is called:

https://github.com/edgurgel/mimic/blob/e525d654568a9362b3cfe05677f6a683de51a063/test/mimic_test.exs#L774-L797

And the test fails! So I think we need something smarter :thinking:  specially to take care of the [transitive](https://github.com/edgurgel/mimic/blob/e525d654568a9362b3cfe05677f6a683de51a063/test/mimic_test.exs#L818) aspect :thinking: 

I'm not sure what is the best approach yet though :thinking:

Reply:
Maybe in this case we want to just do

```
:ets.insert(__MODULE__, {{allowed_pid, module}, owner_pid})
```

?

Reply:
From what I can tell that did work, it is working with my test suite as well and added a test for the case you mentioned